packages/gestalt/src/Dropdown/OptionItem.js

```diff
@@ -104,19 +108,22 @@ const OptionItemWithForwardRef: AbstractComponent<Props, ?HTMLElement> = forward
     [focusStyles.accessibilityOutline]: isFocusVisible,
     [focusStyles.accessibilityOutlineFocusWithin]: isFocusVisible,
     [styles.fullWidth]: true,
-    [styles.pointer]: true,
+    [styles.pointer]: !disabled,
+    [styles.noDrop]: disabled,
   });
 
+  const textColor = disabled ? 'subtle' : 'default';
+
   const optionItemContent = (
     <Flex>
       <Flex direction="column" flex="grow" gap={{ column: 1, row: 0 }}>
         <Flex alignItems="center">
           {children || (
             <Fragment>
-              <Text color="default" inline lineClamp={1} weight={textWeight}>
+              <Text color={textColor} inline lineClamp={1} weight={textWeight} title="">
```

Comment:
I wouldn't do the title="" . Here it doesn't really bother much and it could be useful in some edge cases.

![Screenshot by Dropbox Capture](https://github.com/pinterest/gestalt/assets/10593890/f3416057-6e8b-40b3-ba1b-09503673a3f4)

Reply:
Good callout. I updated to only have this not shown on disabled items which is the larger root cause